packages/graph/hash_graph/lib/graph/src/lib.rs

```diff
@@ -44,3 +44,7 @@ pub mod api;
 pub mod datastore;
 pub mod logging;
 pub mod types;
+
+#[cfg(test)]
+#[path = "../../../tests/testdata/lib.rs"]
```

Comment:
This is very strange 🙃

Reply:
I chose a directory that can be used commonly for all tests.

Reply:
Yeah it's still very weird to see this module definition in a very different place (and level) to the actual module file, as well as it being test only, etc.

I don't have an idea of how to improve it but I don't like it 🙃